torch/_inductor/utils.py

```diff
@@ -78,7 +78,11 @@ def unique(it):
 
 
 def ceildiv(numer: int, denom: int):
-    assert isinstance(numer, int) and isinstance(denom, int)
+    if isinstance(numer, torch.Tensor):
+        numer = numer.item()
+    if isinstance(denom, torch.Tensor):
+        denom = denom.item()
+    # assert isinstance(numer, int) and isinstance(denom, int), f"{numer}: {type(numer)}, {denom}: {type(denom)}"
```

Comment:
I hate this. But I don't really understand how I'm getting a Tensor as argument here. The backtrace looks like:

```
  File "/data/users/ezyang/a/pytorch/torch/_inductor/compile_fx.py", line 250, in run                               return model(new_inputs)                                                                                    
  File "/tmp/torchinductor_ezyang/o4/co44acxg774qgd4loteojd3mm7pnotnp5bdhgqch3zi3kbdp43re.py", line 558, in call
    triton__5.run(buf19, buf22, s2, triton__5_xnumel, 512, grid=grid(triton__5_xnumel), stream=stream0)           File "/data/users/ezyang/a/pytorch/torch/_inductor/triton_ops/autotune.py", line 185, in run                      return launcher(                                                                                            
  File "<string>", line 3, in launcher                                                                          
  File "/data/users/ezyang/a/pytorch/torch/_inductor/triton_ops/autotune.py", line 707, in grid_fn              
    get_grid_dim(xnumel, meta.get("XBLOCK", None)),                                                             
  File "/data/users/ezyang/a/pytorch/torch/_inductor/triton_ops/autotune.py", line 703, in get_grid_dim             return ceildiv(numel, block)                                                                                
  File "/data/users/ezyang/a/pytorch/torch/_inductor/utils.py", line 81, in ceildiv                             
    assert isinstance(numer, int) and isinstance(denom, int), f"ceildiv({numer}, {denom})"                      
AssertionError: ceildiv(3072, 1)                                                                                
TorchDynamo optimized model failed to run because of following error 
```